city_scrapers/spiders/tulok_bocc.py

```
@@ -0,0 +1,150 @@
+                if "BOCC meets" in p or "Monday" in p:
+                    # Clean up whitespace and normalize
+                    self.time_notes = " ".join(p.split())
+                    break
+
+        # Then fetch the events from the API
+        today = date.today()
+        today_str = today.isoformat()
+        year_start = f"{today.year}-01-01"
+        urls = [
+            # past events from year start up to today
+            f"{self.api_base_url}/v1/Events?$filter=startDateTime+ge+{year_start}+and+startDateTime+lt+{today_str}+and+{self.category_filter}&$orderby=startDateTime+desc,+eventName+desc",  # noqa
```

Ruff (0.14.8)

Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)